src/ig/IGExporter.ts

```diff
@@ -1427,6 +1427,76 @@ export class IGExporter {
           dependency.reason = configDependency.reason;
         }
       });
+    } else {
```

Comment:
This PR introduces this `else` block, which means that the whole `updateForR5` function is now a single if-else at the top level. I feel like it might be simpler if instead, the contents of each block were split into their own functions, and the if-else moved into `export`, which is where `updateForR5` is called. Something like this:
```ts
// in export()
if (!isR4(this.config.fhirVersion)) {
  this.updateForR5(); // this would be the if-block contents
} else {
  this.addR5Extensions(); // this would be the else-block contents
}
```
Do you think this makes sense?

Reply:
Ah I like that approach much better! I'll update.

Reply:
Updated in [c3f6501](https://github.com/FHIR/sushi/pull/1189/commits/c3f6501f590435cf43fa93af910e3be923cb1fde).